release-schema.json

```
@@ -98,6 +98,45 @@
             "null"
           ],
           "minLength": 1
+        },
+        "advancedElectronicSignatureRequired": {
+          "title": "Advanced electronic signature required",
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Whether an advanced or qualified electronic signature or seal (as defined in European Parliament and Council Regulation (EU) No 910/2014) is required."
```

Let's leave it up to each jurisdiction to decide the legal definition.

```suggestion
          "description": "Whether an advanced or qualified electronic signature or seal is required."
```